style.css

```diff
@@ -483,3 +483,294 @@ hr {
+    text-align: justify;
+    padding-left: 150px;
+  }
+
+  .img {
+    float: left;
+    width: 684px;
+    height: 448px;
+    padding: 50px;
+    margin-top: 2px;
+    margin-left: 60px;
+  }
+
+  .info-title,
+  .info-detail,
+  .lang {
+    justify-items: right;
+    margin-top: 50px;
+  }
+
+  .btn {
+    float: left;
+    background: #ff6b00;
+    color: #fff;
+    width: 120px;
+    height: 48px;
+    align-items: center;
+    text-align: center;
+    padding: 12px;
+    margin: 30px;
+    border: none;
+    font-family: 'Inter', sans-serif;
+    font-size: 17px;
+    font-weight: bold;
+    line-height: 24px;
+    cursor: pointer;
+    transition-duration: 00.4s;
+  }
+
+  .project-display {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+  }
+
+  .firstproject {
+    background-image: url(media/project.png);
+    opacity: 1;
+  }
+
+  .secondproject,
+  .fifthproject {
+    background-image: url(media/layout2nd.png);
+    opacity: 1;
+  }
+
+  .thirdproject,
+  .sixthproject {
+    background-image: url(media/layout3rd.png);
+    opacity: 1;
+  }
+
+  .forthproject {
+    background-image: url(media/Layout.png);
+    opacity: 1;
+  }
+
+  .contact-form form ul {
+    width: 90%;
+  }
+
+  .firstproject:first-child ul,
+  .firstproject:first-child h3,
+  .firstproject:first-child p {
+    visibility: hidden;
+  }
+
+  .display .btn-dis {
+    display: none;
+  }
+
+  .display:first-child .btn-dis {
+    display: block;
+  }
+
+  /* -------------About Me---------------------- */
+
+  .about-me {
+    background-image: url(media/illustration-about\ me@2x.png);
+    background-position: 100% 15%;
+    background-size: 35%;
+    border: hidden;
+  }
+
+  .about-me-heading,
+  .about-me-content,
+  .btn-about-me {
+    display: flex;
+    text-align: justify;
+    margin-left: 150px;
+    margin-right: 150px;
+  }
+
+  .about-me-heading {
+    font-size: 72px;
+    line-height: 81px;
+  }
+
+  .about-me-content {
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  /* ---------------Programming Language------------ */
+
+  .prg {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    align-items: baseline;
+    margin-bottom: 120px;
+    width: 1024px;
+    margin-left: 200px;
+  }
+
+  /* ----------Contact Form--------------- */
+
+  .contact-form {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: baseline;
+  }
+
+  .contact-form h2 {
+    width: 30%;
+    margin-right: 2%;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 30px;
+    line-height: 45px;
+    text-align: justify;
+  }
+
+  .contact-form form {
+    width: 55%;
+    align-items: flex-start;
+    border-bottom: none;
+  }
+
+  .contact-form .links {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    align-items: center;
+    margin-top: 100px;
+    border-top: 1px solid #dfe1e6;
```

Comment:
- [ ] I can't tell were the input is, please give them borders

Reply:
I followed the same template as figma. I'll give border.